src/Database/PostgreSQL/PQTypes/Utils.hs

```diff
@@ -3,11 +3,31 @@ module Database.PostgreSQL.PQTypes.Utils (
   , raw
   , runQuery_
   , runQuery01
+  , runQuery01OrElse
```

Comment:
There's so many variants. Do we need all of these?

runSQLAndFetch* are redundant because you can fetch afterwards, right?